test/propEqual.js

```diff
@@ -0,0 +1,708 @@
+QUnit.module( "propEquiv" );
```

Comment:
Would "equiv.props" be fine? It would be the same pattern used in test/deepEqual.js